src/popup/utils/constants.ts

```diff
@@ -48,6 +48,9 @@ export const STUB_CALLDATA = 'cb_AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAACDJfUr
 export const STUB_NONCE = 10000;
 export const MAX_UINT256 = new BigNumber(2).exponentiatedBy(256).minus(1);
 export const STUB_TOKEN_CONTRACT_ADDRESS = 'ct_T6MWNrowGVC9dyTDksCBrCCSaeK3hzBMMY5hhMKwvwr8wJvM8';
+export const STUB_CURRENCY = [{
```

Comment:
In my opinion we should keep the mocked data required for the tests in separate constants file located somewhere inside of the `tests/` directory.

Or maybe we can move this into the `src/popup/utils/config.js` file which is mostly used by the test files.